expath-pkg.xml.tmpl

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package xmlns="http://expath.org/ns/pkg" name="http://exist-db.org/apps/public-repo" abbrev="@project.name@" version="@project.version@" spec="1.0">
     <title>eXist-db Public Application Repository</title>
-    <dependency processor="http://exist-db.org" semver-min="5.3.0"/>
+    <dependency processor="http://exist-db.org" semver-min="6.0.0"/>
```

Comment:
why was the minimum required version raised to 6?

Reply:
failing smoke tests, I didn't have time to finish this, but wanted to get rid of some noise on the way there.